tests/Unit/Evolution/DiscontinuousGalerkin/Actions/Test_ComputeTimeDerivative.cpp

```diff
@@ -12,6 +12,11 @@ void test_wrapper() {
   test<system_type, use_boundary_correction, 1>();
   test<system_type, use_boundary_correction, 2>();
   test<system_type, use_boundary_correction, 3>();
+  if constexpr (use_boundary_correction == UseBoundaryCorrection::Yes) {
+    test<system_type, use_boundary_correction, 1>();
+    test<system_type, use_boundary_correction, 2>();
+    test<system_type, use_boundary_correction, 3>();
+  }
```

Comment:
I don't understand this diff. Am I missing something, or are the three lines in the if constexpr identical to the three lines above it? What is the intent?

Reply:
I think this is an artifact of previous refactorings of the test. I also can't tell why the heck this is happening :P